docs/source/getting-started/tabular-data/index.rst

```diff
@@ -0,0 +1,103 @@
+   * - ``HWUSUAL``
+     - Number of Hours Per Week Usually Worked
+     - 00: Usual hours cannot be given because hours worked vary considerably from week to week or from month to month; 01-98: Number of hours usually worked in the main job; 99: Not applicable; blank: No answer
+   * - ``QUARTER``
+     - Fixed Reference Quarter
+     - Single Quarter
+   * - ``YEAR``
+     - Fixed Reference Year
+     - Single Year
+
+Compositor Overview
+-------------------
+The compositor is the foundation of our differentially private queries. It essentially takes in our data and our specifications for the queries that we would like to run. At this point, we won't be directly referencing our data again and we could theoretically delete it! 
+
+.. code-block:: python
+
+   context = dp.Context.compositor(
+       data=df,
+       privacy_unit=dp.unit_of(contributions=36),
+       privacy_loss=dp.loss_of(epsilon=1.0),
+       split_evenly_over=10,
+       margins={
+           ("YEAR", ): dp.Margin(max_partition_length=60_000_000, max_partition_contributions=4),
+           ("YEAR", "QUARTER",): dp.Margin(max_partition_length=60_000_000, max_partition_contributions=1),
+           (): dp.Margin(max_partition_length=60_000_000),
+       },
+   )
+
+**Parameters**:
+
+* *privacy_unit:* How many rows each individual or entity of interest contributes to our data frame. In this case, we are analyzing the data from across 13 years and each year has 4 quarters. Therefore, the unit of privacy is 36. If we were to analyze a particular quarter in a particular year, the unit of privacy would be 1 since each individual would be represented once. 
+
+* *privacy_loss:* This parameter determines how much privacy we want to preserve. If ε is small, we will have more privacy but worse data accuracy. ε can range from 0 to infinity, but 1 is usually a standard. 
+
+* *split_evenly_over:* This is the number of queries you want to distribute your privacy loss over. For now we specified 10 to explore the API but in the final versions of your code, this parameter will be picked more carefully. 
+
+* *margins:* Margins capture the variables of interest in your analysis. This can include variables that you may want to group by or apply differential privacy techniques to. 
+    * *max_partition_length:* The upper bound on how many records (individuals in this case) can be in one partition. If you do not know the size of your dataset, this can be an upper bound on the population represented in your dataset. The population of France was about 60 million in 2004 so that's our maximum partition length. Source: `World Bank <https://datatopics.worldbank.org/world-development-indicators/>`_. 
```

Comment:
What are we partitioning and why?

Reply:
i added this: lmk what to put and how else to elaborate! 

    Partitioning the data set may be useful to increase scalability and utility.

Reply:
Like is partitioning part of this particular application or is it something specifically needed for the library. Just mention the purpose of partitioning.